app/models/concerns/entity_access.rb

```diff
@@ -89,6 +172,63 @@ def editable_role
     self.class.viewable_permission
   end
 
+  private def viewable_user_group_name
+    "#{name} [viewable]"
+  end
+
+  private def editable_user_group_name
+    "#{name} [editable]"
+  end
+
+  # NOTE: this only needs to be run once, and only if the installation was created
+  # prior to release-151
```

Comment:
maybe we should add a todo-or-die to remove dead code once everyone has migrated?
```suggestion
  # prior to release-151
  TodoOrDie('Remove fix_entity_associations', by: '2025-04-01')
```

Or alternatively move the method into the migration where we will purge it eventually